### ISSUE TITLE
Energy-based (EBGAN)

### ISSUE THREAD

**Issue by Aiden-Jeon** [Aiden-Jeon/pytorch_gan#3]
[paper](https://arxiv.org/pdf/1609.03126.pdf)

**Comment by Aiden-Jeon** [comment]
Discriminator
![image](https://user-images.githubusercontent.com/33924485/77647823-7538c680-6faa-11ea-9095-a24f2d3a1a76.png)

**Comment by Aiden-Jeon** [comment]
![image](https://user-images.githubusercontent.com/33924485/77657359-8983c000-6fb8-11ea-9447-728c4a855ab2.png)

**Comment by Aiden-Jeon** [comment]
- margin
  - MNIST: 10
  - LSUN : 80
  - CelebA : 20
  - ImageNet 
    - 128 x 128 : 40
    - 256x 256 : 80

**Comment by Aiden-Jeon** [comment]
CIFAR10 
learning rate = 2e-4 
margin = 20
